put_get_test.go

```diff
@@ -99,9 +99,10 @@ func TestPercentage(t *testing.T) {
 		{14, 28, 0.5},
 	}
 	for _, test := range testcases {
-		if percent(test.seen, test.size) != test.expected {
+		spp := &snowflakeProgressPercentage{}
```

Comment:
Do we need a pointer here? Decide it and close my comment and you are good to go!